basis/listener/listener.factor

```diff
@@ -120,7 +125,7 @@ t error-summary? set-global
 
     [
         read-quot [
-            '[ datastack _ with-datastack ]
+            [ '[ datastack _ with-datastack ] with-ctrl-break ] curry
```

Comment:
@bjourne @erg @mrjbq7 
Am I right that this line can be simpler, like so:
'[ [ datastack _ with-datastack ] with-ctrl-break ]